runtime/compiler/z/codegen/S390J9CallSnippet.cpp

```diff
@@ -1718,3 +1720,183 @@ TR::S390InterfaceCallReadOnlySnippet::getLength(int32_t estimatedSnippetStart)
            + sizeof(int32_t)  /*RIP offset to PICSlot data address*/
            + sizeof(int32_t);  /*RIP offset to instruction in mainline*/
    }
+ 
+uint8_t *
+TR::S390J9CallSnippetRX::emitSnippetBody()
+   {
+   uint8_t *cursor = cg()->getBinaryBufferCursor();
+   TR::SymbolReference *callSymRef = getRealMethodSymbolReference();
+   TR::Node *callNode = getNode();
+   getSnippetLabel()->setCodeLocation(cursor);
+   cursor = S390flushArgumentsToStack(cursor, getNode(), getSizeOfArguments(), cg());
+
+   //Check if the call is unresolved
+   if (callSymRef->isUnresolved() || (cg()->comp()->compileRelocatableCode() && !cg()->comp()->getOption(TR_UseSymbolValidationManager)))
+      {
+      typedef J9::Z::PrivateLinkage::ccUnresolvedStaticSpecialData ccUnresolvedStaticSpecialData;
+      ccUnresolvedStaticSpecialData *ccDataAddress = reinterpret_cast<ccUnresolvedStaticSpecialData *>(callSnippetCCDataAddress);
+      ccDataAddress->snippetOrCompiledMethod = reinterpret_cast<intptr_t>(getSnippetLabel()->getCodeLocation());
+      // LGRL  rEP, @(CCUnresolvedStaticOrSpecialData + ramMethod)
+      intptr_t instrAddr = reinterpret_cast<intptr_t>(cursor);
+      *cursor = static_cast<uint8_t>(0xC4);
```

Comment:
Adding a new line right before a comment can go a long way to improve readability and emphasize a logical block of code. Otherwise it just looks like one giant blob of code.

Reply:
That makes sense, sorry for the mess. Tried to add more new lines to make call more readable in https://github.com/eclipse/openj9/compare/9a9d9b37daeb983c8d4e295fcc785afeecde5d78..de725983cd06ff96468a76b30321d3f8bb48caea